browser/ephemeral_storage/BUILD.gn

```diff
@@ -16,7 +16,10 @@ source_set("ephemeral_storage") {
 if (!is_android) {
   source_set("ephemeral_storage_tests") {
     testonly = true
-    sources = [ "ephemeral_storage_browsertest.cc" ]
+    sources = [
+      "ephemeral_storage_browsertest.cc",
+      "ephemeral_storage_qa_browsertest.cc",
```

Comment:
I don't think we need `qa` here. We want the tests to be arranged logically and I think it's better to put them all together so it's easier to see what is covered

Reply:
what should I call these? I can't call it just `ephemeral_storage_browsertest` since we already have that.